components/qt5/Makefile

```diff
@@ -0,0 +1,124 @@
+#
```

Comment:
Nope just me messing with git. Was an earlier experiment. Need to move it to other branch.